### PR TITLE
W3HW6 tests for model

### PR DESCRIPTION
**Pull request by yevhen-k**
Hi @truskovskiyk 
Here is tests for model. With later commits I'll provide a CI test with git actions

### CODE REVIEW

**Comment by yevhen-k** [comment]
@truskovskiyk 
Tests are green :) please, check it

File: week-3-tests/train.py [hunk 1]
```diff
@@ -44,16 +44,17 @@ def train(output_dir="./weights",
...
     # Load data
     imdb = get_dataset(dataset)
 
     # Create a smaller training dataset for faster training times
     small_train_dataset, small_test_dataset = prepare_dataset(imdb, n_train=n_train, n_test=n_test, seed=seed)
     
-    print(small_train_dataset[0])
-    print(small_test_dataset[0])
+    # print(small_train_dataset[0])
+    # print(small_test_dataset[0])
```
Comment (truskovskiyk) [inline week-3-tests/train.py:57]: remove commented code

File: week-3-tests/test_model.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+class TestModel:
+
+    def test_invariance_verbs(self, get_pipeline):
+        model = get_pipeline
+        tokens = ["clarified", "reveiled"]
+        texts = [f"The movie is awesome and it {word} the meaning of life" for word in tokens]
+        predicts = model(texts)
+        assert all([predict["label"] == predicts[0]["label"] for predict in predicts])
+
+    def test_invariance_numbers(self, get_pipeline):
```
Comment (truskovskiyk) [inline week-3-tests/test_model.py:24]: nice nice!

**Review by truskovskiyk — APPROVED** [review]
@yevhen-k amazing job!

**Pull request merged**